src/index.tsx

```diff
@@ -476,12 +476,6 @@ function CellRendererComponent<T>(props: CellRendererProps<T>) {
   );
 }
 
-declare module "react" {
-  function forwardRef<T, P = {}>(
-    render: (props: P, ref: React.Ref<T>) => React.ReactNode | null
-  ): (props: P & React.RefAttributes<T>) => JSX.Element | null;
-}
-
```

Comment:
Could you describe a bit why this PR gets rid of this? I believe it's necessary to get Typescript to work right for forward refs.